README.md

```diff
@@ -264,5 +266,28 @@ Golang agent supports the following dynamic configurations.
 |:-----------------:|:----------------------------------------------------------------------------------------:|:--------------------:|
 | agent.sample_rate | The percentage of trace when sampling. It's `[0, 1]`, Same with `WithSampler` parameter. |         0.1          |
 
+## Process Status Hook
+
+This feature is used in cooperation with the [skywalking-rover](https://github.com/apache/skywalking-rover) project.
+
+When go2sky keeps alive with the backend, it would write a metadata file to the local (temporary directory) at the same time, which describes the information of the current process.
+The rover side scans all processes, find out which process contains this metadata file. Finally, the rover could collect, profiling, with this process.
```

Comment:
```suggestion
When go2sky keeps alive with the backend, it would write a metadata file to the local (temporary directory) at the same time, which provides the information of the current process.
The rover would detect this file in order to identify this Golang service, and tags it `ebpf-profiling-able`.
```

As a doc, we don't need to describe how codes work, they are open sourced :) We need to tell why it works in this way.

Reply:
`ebpf-profiling-able`? Do you mean to add this tag to the process labels?

Reply:
I mean logically, this is what this is designed for. Nothing about label.